### PR TITLE
Feat 012

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -9,12 +9,13 @@ import io.appwrite.services.Storage
...
 import java.io.File
+import kotlin.math.min
 
 
 val client = Client()
     .setEndpoint("https://demo.appwrite.io/v1")
-    .setProject("608fa1dd20ef0")
-    .setKey("YOUR_KEY")
+    .setProject("playgrounds")
+    .setKey("98d6dc820108f1bbcbf3a29c4d8de81a7a044356455938743ee71b97af7e528212899d247d8cd759ebd7219e836a768acf78dd8de40954b6e6829e9ec90fc6eb4ea024f13662530c48b93679512abf5224396ceed6ac4b32408fed3b36db668c81ceb3bf09e07ebd8d0d182754d879453436bb5ced561c9a9d9ebd9c5aa83af9")
```
Comment (abnegate) [inline src/main/kotlin/Main.kt]: We should remove the key from here

Comment (christyjacob4) [inline src/main/kotlin/Main.kt]:
```suggestion
    .setProject("playgrounds")
    .setKey("YOUR API KEY")
```

File: src/main/kotlin/Main.kt [hunk 2]
```diff
@@ -9,12 +9,13 @@ import io.appwrite.services.Storage
 import io.appwrite.services.Users
 import kotlinx.coroutines.runBlocking
 import java.io.File
+import kotlin.math.min
 
 
 val client = Client()
     .setEndpoint("https://demo.appwrite.io/v1")
```
Comment (christyjacob4) [inline src/main/kotlin/Main.kt]:
```suggestion
    .setEndpoint("https://localhost/v1")
```

File: src/main/kotlin/Main.kt [hunk 3]
```diff
@@ -9,12 +9,13 @@ import io.appwrite.services.Storage
...
 import java.io.File
+import kotlin.math.min
 
 
 val client = Client()
-    .setEndpoint("https://demo.appwrite.io/v1")
-    .setProject("608fa1dd20ef0")
-    .setKey("YOUR_KEY")
+    .setEndpoint("https://localhost/v1")
+    .setProject("playgrounds")
```
Comment (christyjacob4) [inline src/main/kotlin/Main.kt]:
```suggestion
    .setProject("YOUR PROJECT ID")
```

**Pull request merged**